lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -185,10 +200,21 @@ def allowed_to_execute?
         agree("Are you sure you would like to restore the database? (Y/N): ")
       end
 
+      def file_options
+        @file_options ||= I18n.t("database_admin.menu_order").each_with_object({}) do |file_option, h|
+          # special anonymous ftp sites
+          if file_option =~ %r{^ftps?://([^/]*)/}
```

Comment:
This might be easier to read without the regex. None of the other file options are uri formatted right? How about something like this:

```ruby
if %w(ftp ftps).include?(URI(file_option).scheme)
```

Then we can also avoid the global match variable by using the host in the URI.